docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,808 @@
+# Working with the CHIP Tool
+
+The CHIP Tool (`chip-tool`) is a Matter controller implementation that allows to
+commission a Matter device into the network and to communicate with it using
+Matter messages, which may encode Data Model actions, such as cluster commands.
+The tool also provides other utilities specific to Matter, such as parsing of
+the setup payload or performing discovery actions.
+
+<hr>
+
+-   [Source files](#source)
+-   [Building and running the CHIP Tool](#building)
+-   [Using the CHIP Tool for Matter device testing](#using)
+-   [Supported commands and options](#commands)
+
+<hr>
+
+<a name="source"></a>
+
+## Source files
+
+You can find source files of the CHIP Tool in the `examples/chip-tool`
+directory.
+
+> **Note:** The CHIP Tool currently only supports commissioning and remembering
```

Comment:
I think it is not true. Chip tool is able to commission multiple devices and control them but not in parallel. @Damian-Nordic do you agree with me?